apps/docs/src/components/code/PopupLayout.tsx

```diff
@@ -80,9 +80,7 @@ export const PopupLayout = ({ scope, code }: PopupLayoutProps) => {
       />
 
       {/* Preview Section */}
-      <div className="p-md flex items-center justify-center">
-        <div>{element}</div>
-      </div>
+      <div className="p-md flex items-center justify-center">{element}</div>
```

Comment:
This will affect other samples. The parent of the samples must be "clean", without custom layouts

Please revert.